lib/Agrammon/Web/Routes.pm6

```diff
@@ -5,100 +5,67 @@ use Cro::HTTP::Router;
 use Agrammon::Web::Service;
 use Agrammon::Web::SessionUser;
 
-
 sub routes(Agrammon::Web::Service $ws) is export {
+    subset LoggedIn of Agrammon::Web::SessionUser where .logged-in;
+
+    # Fix for Agrammon
+    # my $root = $app ~~ Easii::Application::Development ?? 'frontend/source-output' !! 'qx-build';
```

Comment:
Maybe don't mention Easii? :-) Also, I'm curious if `$root` is over-used in the static content? It seems primarily relevant for the scripts, but maybe not the HTML?

Reply:
Removed Easii ... would like to keep `$root` as is until I have the frontend build (for both local and deployed) figured out.